### PR TITLE
Wildfire, intezer file commands - args additions and error grace handling

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/16113
fixes: https://github.com/demisto/etc/issues/16642

Also added the 'is array' option for intezer that was missing with correlating handling of the entries.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [x] Documentation https://github.com/demisto/etc/issues/16640, https://github.com/demisto/etc/issues/16576
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-Intezer.yml [hunk 1]
```diff
@@ -232,25 +235,36 @@ script:
...
 
     switch (command) {
         case 'test-module':
             // send is available
             return sendRequest('GET', IS_AVAILABLE_URL).obj.is_available;
+
         case 'file':
-            return analyzeSHA256(args.file, args.codeItuemType, args.maxRetries, args.delay);
+            if ((args.sha256 && args.file) || (!args.sha256 && !args.file)) {
+                throw 'Specify only 1 of the arguments: file, sha256';
```
Comment (anara123) [inline Integrations/integration-Intezer.yml]: return error entry or catch the exception and return error entry and to the log. we dont want to print stack to the warroom

File: Integrations/integration-Intezer.yml [hunk 2]
```diff
@@ -260,9 +274,12 @@ script:
   - name: file
     arguments:
     - name: file
-      required: true
       default: true
       description: Hash of the file to query. Supports SHA256
+      isArray: true
```
Comment (anara123) [inline Integrations/integration-Intezer.yml]: use isArray only if the API support array

File: Integrations/integration-Intezer.yml [hunk 3]
```diff
@@ -125,7 +127,7 @@ script:
     var analyzeSHA256 = function(hash, codeItemType, maxRetries, delay) {
         // validate file is indeed sha256
         if (!SHA_256_REGEXP.test(hash)) {
-            var warning = ERROR_PREFIX + 'File hash is not valid sha256.\nIntezer file hash reputation supports only sha256 hash.\n\n';
+            var warning = 'Intezer file hash reputation does not support MD5 and SHA1 hashes.\n';
```
Comment (anara123) [inline Integrations/integration-Intezer.yml]: The warning should be that Intezer supports only sha256

File: Integrations/integration-Wildfire.yml [hunk 4]
```diff
@@ -266,26 +267,38 @@ script:
...
-    function hashArgsHandler(md5, hash, file) {
-        if(file) {
-            if( MD5_REGEX.test(file) || SHA256_REGEX.test(file)){
-                return [file];
-            } else {
-                throw 'Invalid hash. Only SHA256 and MD5 are supported.';
+    function hashArgsHandler(hash, file, md5, sha256) {
+        // file/md5/sha256 are used in file command
+        if ((file && !md5 && !sha256) || (!file && md5 && !sha256) || (!file && md5 && !sha256)) {
+            if (file) {inputs = argToList(file);}
```
Comment (anara123) [inline Integrations/integration-Wildfire.yml]: dont use one line code

File: Integrations/integration-Wildfire.yml [hunk 5]
```diff
@@ -266,26 +267,38 @@ script:
...
+            else if (md5) {inputs = argToList(md5);}
+            else {inputs = argToList(sha256);}
```
Comment (anara123) [inline Integrations/integration-Wildfire.yml]: define inputs in the beginning of the code

File: Integrations/integration-Intezer.yml (hunk 2 above)
Comment (bakatzir) [reply Integrations/integration-Intezer.yml]: mutually decided to keep it

**Comment by bakatzir** [comment]
last fail is on lastline quota, will re run tomorrow

**Pull request merged**